core/trino-main/src/test/java/io/trino/operator/scalar/TestMathFunctions.java

```diff
@@ -1354,18 +1355,26 @@ public void testWidthBucket()
         assertInvalidFunction("width_bucket(3.14E0, -1, infinity(), 3)", "second bound must be finite");
     }
 
-    @Test(expectedExceptions = TrinoException.class, expectedExceptionsMessageRegExp = "Bucket for value Infinity is out of range")
+    @Test
     public void testWidthBucketOverflowAscending()
     {
-        functionAssertions.tryEvaluate("width_bucket(infinity(), 0, 4, " + Long.MAX_VALUE + ")", DOUBLE);
-        functionAssertions.tryEvaluate("width_bucket(CAST(infinity() as REAL), 0, 4, " + Long.MAX_VALUE + ")", DOUBLE);
+        assertThatThrownBy(() -> functionAssertions.tryEvaluate("width_bucket(infinity(), 0, 4, " + Long.MAX_VALUE + ")", DOUBLE))
+                .isInstanceOf(TrinoException.class)
+                .hasMessage("Bucket for value Infinity is out of range");
+        assertThatThrownBy(() -> functionAssertions.tryEvaluate("width_bucket(CAST(infinity() as REAL), 0, 4, " + Long.MAX_VALUE + ")", DOUBLE))
+                .isInstanceOf(TrinoException.class)
+                .hasMessage("Bucket for value Infinity is out of range");
     }
 
-    @Test(expectedExceptions = TrinoException.class, expectedExceptionsMessageRegExp = "Bucket for value Infinity is out of range")
+    @Test
     public void testWidthBucketOverflowDescending()
     {
-        functionAssertions.tryEvaluate("width_bucket(infinity(), 4, 0, " + Long.MAX_VALUE + ")", DOUBLE);
-        functionAssertions.tryEvaluate("width_bucket(CAST(infinity() as REAL), 4, 0, " + Long.MAX_VALUE + ")", DOUBLE);
+        assertThatThrownBy(() -> functionAssertions.tryEvaluate("width_bucket(infinity(), 4, 0, " + Long.MAX_VALUE + ")", DOUBLE))
+                .isInstanceOf(TrinoException.class)
+                .hasMessage("Bucket for value Infinity is out of range");
+        assertThatThrownBy(() -> functionAssertions.tryEvaluate("width_bucket(CAST(infinity() as REAL), 4, 0, " + Long.MAX_VALUE + ")", DOUBLE))
+                .isInstanceOf(TrinoException.class)
+                .hasMessage("Bucket for value Infinity is out of range");
```

Comment:
I couldn't find the reason not to use `assertInvalidFunction()` method. Could you rewrite these 4 assertions in this class?